Atomic/syscontainers.py

```diff
@@ -743,6 +743,9 @@ def has_system_container_image(self, img):
         if not repo:
             return False
         try:
+            for i in self.get_system_images(get_all=True):
+                if i['Id'].startswith(img):
+                    return True
```

Comment:
since the patch for `verify` now stores the images in `OSTREE_OCIIMAGE_PREFIX` with the "oci-sha256", is there a clean way to convert the image id to the oci sha256?

Otherwise, would:
```
        try:
            imagebranch = SystemContainers._get_ostree_image_branch(img)
            return repo.resolve_rev(imagebranch, False)[0]
        except: #pylint: disable=bare-except
            for i in self.get_system_images(get_all=True):
                if i['Id'].startswith(img):
                    return True
            return False
```
be an acceptable solution?